WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -24,21 +24,38 @@
 - [ ] Verify that you can delete a bookmark in edit mode
 - [ ] Verify that you can delete a bookmark folder with bookmarks inside
 - [ ] Verify adding a bookmark domain, subpaths is retained and you are successfully able to visit the domain subpath in a new tab
+- [ ] Verify adding bookmark for many open tabs works. Ensured opening all bookmarks in `Saved tabs` folder will load all the tabs
+- [ ] Verify import/export bookmarks works
 
 ## Favourites
 
 - [ ] Verify editing favourites and changing their URLs updates the favicons accordingly
 - [ ] Verify that you can remove favourites
 - [ ] Verify that you can add new favourites from the `Share with...` menu
+- [ ] Verify adding Favorites Widget to home screen and ensure favorites are updated on the widget accordingly
 
 ## Context menus
 
 - [ ] Make sure context menu items in the URL bar work
 - [ ] Make sure context menu items on content work with no selected text
 - [ ] Make sure context menu items on content work with selected text
 - [ ] Make sure context menu items on content work inside an editable control (input, textarea, or contenteditable)
-- [ ] Verify you can open links in a new tab or private tab. Ensure you can open links in background tabs.
-
+- [ ] Verify you can open links in a new tab or private tab. Ensure you can open links in background tabs
+- [ ] Verify all actions work in context menu when long pressing on `+` in the tool bar
+- [ ] Verify all actions work in context menu when long pressing on tab icon
+- [ ] Verify `Recently Closed Tabs` is opened when long pressing `+` icon in tab view
+
+## Navigation
+
+- [ ] Verify search works from the search box on NTP
+- [ ] Verify searching using voice search icon
```

Comment:
Suggestion
`Verify able to perform voice search by pressing the microphone icon in URL bar`